Mage.Sets/src/mage/sets/Battlebond.java

```diff
@@ -9,23 +9,27 @@
  * @author TheElk801
  */
 public final class Battlebond extends ExpansionSet {
-
+    
     private static final Battlebond instance = new Battlebond();
-
+    public int i;
```

Comment:
What is this variable for?

Reply:
Oops, removed it. Both the variable and the change to public were leftovers from the previous iteration. At first I was going to implement a pattern searching in the set class but after looking closely at the code, decided against it for obvious reasons